device-server/src/main/kotlin/com/badoo/automation/deviceserver/ios/simulator/SimulatorPermissions.kt

```diff
@@ -56,4 +64,33 @@ class SimulatorPermissions(
             throw(SimulatorError("Failed to update type $type for $this"))
         }
     }
+
+    private val appleSimUtils = "/usr/local/bin/applesimutils"
+
+    @Suppress("UNUSED_PARAMETER")
+    private fun setNotificationsPermission(bundleId: String, value: PermissionAllowed) {
+        // Setting notifications permission is disallowed as it results in SpringBoard restart
+        // which breaks WebDriverAgent. Restarting SpringBoard and WebDriverAgent will take to much time.
+        throw RuntimeException("Setting notifications permission is not supported")
+    }
+
+    private fun setLocationPermission(bundleId: String, value: PermissionAllowed) {
```

Comment:
1. rename 'value' to permissionAllowed.
2. use strings from enum

```
        val permissionValue = when (permissionAllowed) {
            Always, Inuse, Never, Unset -> permissionAllowed.value
            else -> throw IllegalArgumentException("")
        }
```

Reply:
I don't think we should use strings from enum here. Enum string values are DTOs and used on the client side, while strings we use here are internal implementation specific to applesimutils. And its just a coincidence that applesimutils use the same values as our DTO.